client/cmd/internal/logger/color.go

```diff
@@ -0,0 +1,35 @@
+package logger
+
+import (
+	"fmt"
+
+	"github.com/muesli/termenv"
+	"github.com/odpf/salt/term"
+)
+
+var (
+	// ColoredSuccess format message with color for success
+	ColoredSuccess = fmt.Sprintf
+	ColoredNotice  = fmt.Sprintf
+	ColoredError   = fmt.Sprintf
+
+	tp          = termenv.EnvColorProfile()
+	ColorYellow = tp.Color("#FFAF00")
+	ColorWhite  = tp.Color("#FFFFFF")
+	ColorRed    = tp.Color("#D70000")
+)
+
+func InitializeColor() {
```

Comment:
is it intended to introduce this and not using the existing color function that being used in other commands?

Reply:
no, i'll check the existing implementation

Reply:
removed the file